SQL/0000-00-00-schema.sql

```diff
@@ -754,6 +757,39 @@ CREATE TABLE `mri_protocol_checks` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+-- ********************************
+-- BIDS tables
+-- ********************************
+
+CREATE TABLE `BIDS_category` (
```

Comment:
table names should be all lowercase snake case according to the LORIS SQL standard (https://github.com/aces/Loris/blob/minor/docs/SQLModelingStandard.md)

Reply:
I checked this document, and it says that exceptions are allowed for MRI acronyms, so I assumed BIDS is an allowed acronym, non?

Reply:
changed it to `bids` instead in the last commit